domain/consensus/processes/dagtraversalmanager/anticone.go

```diff
@@ -48,6 +50,13 @@ func (dtm *dagTraversalManager) AnticoneFromBlocks(stagingArea *model.StagingAre
 			return nil, err
 		}
 
+		// We count the number of blocks in past(tips) \setminus past(blockHash).
+		// We don't use `len(visited)` since it includes some maximal blocks in past(blockHash) as well.
+		traversalCounter++
+		if maxTraversalAllowed > 0 && traversalCounter > maxTraversalAllowed {
+			return nil, model.ErrReachedMaxTraversalAllowed
```

Comment:
The best practice is to wrap errors with `errors.Wrapf`. When you do that, it adds stack information to the error